<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplyDelegatePage</class>
 <widget class="QWidget" name="ApplyDelegatePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>27</y>
     <width>215</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Proxy account upgrade</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>28</y>
     <width>500</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(127, 127, 127);</string>
   </property>
   <property name="text">
    <string>After upgrading to an agent account, you not only have the function of a registered 
user, but also have the right to vote;  n and when your ranking into the top 99, you 
will get the right to block.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(40,40,40);</string>
   </property>
   <property name="text">
    <string>To be upgraded account</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(40,40,40);</string>
   </property>
   <property name="text">
    <string>To be upgraded address</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(40,40,40);</string>
   </property>
   <property name="text">
    <string>Account balance</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>200</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 96, 82);</string>
   </property>
   <property name="text">
    <string>Insufficient balance</string>
   </property>
  </widget>
  <widget class="QComboBox" name="accountComboBox">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>150</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="addressLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>520</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(134, 134, 134);</string>
   </property>
   <property name="text">
    <string notr="true">GOPSSDDSWEXsdfsdffdfdfd</string>
   </property>
  </widget>
  <widget class="QLabel" name="balanceLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>221</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(90, 119, 197);</string>
   </property>
   <property name="text">
    <string notr="true">0 GOP</string>
   </property>
  </widget>
  <widget class="QToolButton" name="upgradeBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>440</y>
     <width>120</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{background-color:#469cfc;color:#ffffff;border:0px solid rgb(64,153,255);border-radius:3px;}
QToolButton:hover{background-color:#62a9f8;}
QToolButton:disabled{background-color:#cecece;}</string>
   </property>
   <property name="text">
    <string>Upgrade</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>371</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#282828;&quot;&gt;Account need &lt;/span&gt;&lt;span style=&quot; font-weight:600; color:#ff503f;&quot;&gt;1221.83GOP &lt;/span&gt;&lt;span style=&quot; font-weight:600; color:#282828;&quot;&gt;balance to upgrade&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>295</y>
     <width>321</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(134, 134, 134);</string>
   </property>
   <property name="text">
    <string>Actual cost is 1221.828181（Include 0.01 fee）</string>
   </property>
  </widget>
  <widget class="QLabel" name="wrongLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>343</y>
     <width>300</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 96, 82);</string>
   </property>
   <property name="text">
    <string>The account has been used, please choose another.</string>
   </property>
  </widget>
  <widget class="QLabel" name="correctLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>343</y>
     <width>350</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(103,182,103);</string>
   </property>
   <property name="text">
    <string>Account is available, once set, we cant change it.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressNameLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>343</y>
     <width>90</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(40,40,40);</string>
   </property>
   <property name="text">
    <string>Upgrade account</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel2">
   <property name="geometry">
    <rect>
     <x>187</x>
     <y>376</y>
     <width>600</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(134, 134, 134);</string>
   </property>
   <property name="text">
    <string>Only lowercase letters and numbers are allowed and must begin with lowercase, This account will be
permanently registered on the Goopal block and can not be deleted and modified.</string>
   </property>
  </widget>
  <widget class="QLabel" name="wrongLabel2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>343</y>
     <width>200</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 96, 82);</string>
   </property>
   <property name="text">
    <string>More than 63 characters</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
